scripts/entrypoint.sh

```diff
@@ -35,6 +35,15 @@ log_in_json()
     printf '{"level":"%s","ts":"%s","caller":"%s","msg":"%s"}\n' "$LOGTYPE" "$TIMESTAMP" "$FILENAME" "$MSG"
 }
 
+validate_env_var()
+{
+    log_in_json info "Validating env variables ..."
+    if [[ "$AWS_VPC_K8S_PLUGIN_LOG_FILE" == "stdout" ]]; then
+        log_in_json error "Plugin log file cannot be set to stdout"
```

Comment:
can we rephrase this to, AWS_VPC_K8S_PLUGIN_LOG_FILE env var cannot be stdout instead ?

Reply:
Sure will do it.